ambry-frontend/src/main/java/com.github.ambry.frontend/FrontendMetrics.java

```diff
@@ -101,6 +103,9 @@
   public final Histogram securityServiceProcessResponseTimeInMs;
   // AmbryIdConverter
   public final Histogram idConverterProcessingTimeInMs;
+  // GetPeersHandler
+  public final Histogram getPeersSecurityProcessingTimeInMs;
```

Comment:
Sorry, this isn't the callback processing time. I guess this is `SecurityRequestTime`, right?